src/gui/main.cpp

```diff
@@ -207,6 +207,43 @@ CommandLineOptions parseOptions(const QStringList &arguments)
     return out;
 }
 
+/**
+ * Maybe a newer version of the client was used with this config file:
+ * if so, backup, confirm with user and remove the config that can't be read.
+ */
+bool configVersionMigration()
```

Comment:
we backup and don't migrate? So I guess we should update the name.

Reply:
changed the comment: only migrate when updating

Reply:
renamed it to `checkClientVersion()`